packages/now-next/src/index.ts

```diff
@@ -612,6 +623,7 @@ export const build = async ({
     });
 
     const pageKeys = Object.keys(pages);
+    const hasLambdas = !staticPages['_errors/404'] || pageKeys.length > 1;
```

Comment:
Probably good to add a comment saying the `> 1` is because if there is no static 404 `/_error` will be a lambda.

Reply:
Added